GPU/Common/TextureCacheCommon.cpp

```diff
@@ -2315,9 +2315,9 @@ void TextureCacheCommon::ApplyTextureFramebuffer(VirtualFramebuffer *framebuffer
 		bool needsDepthXSwizzle = depthUpperBits == 2;
 
 		int depalWidth = framebuffer->renderWidth;
```

Comment:
Hm, does this mean that renderWidth is wrong too?  Or else, it's wrong inside the `needsDepthXSwizzle` case.  But it seems right that we would care about the actual framebuffer width.

-[Unknown]

Reply:
Oh yeah, it does mean that. I think we just got lucky before.

Reply:
I think actually it's fine as-is - although, if the framebuffer somehow got even wider than 512 then that rounduptopowerof2 inside the if would round up to way too big.

But technically in the normal case, I guess we don't actually need to depal beyond fb->width although we still want to size the depal texture the same as the source texture for simplicity. Could use an UV range to arrange that.

Anyway I tested the games that use this swizzle, and they're fine...

Reply:
Okay, the two formulas just feel inconsistent but I guess it's probably fine...

-[Unknown]